ts/features/wallet/onboarding/common/searchBank/SearchStartComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import { View } from "native-base";
+import * as React from "react";
+import { fromNullable } from "fp-ts/lib/Option";
+import I18n from "../../../../../i18n";
+import { Body } from "../../../../../components/core/typography/Body";
+import { H1 } from "../../../../../components/core/typography/H1";
+import { H4 } from "../../../../../components/core/typography/H4";
+import { Link } from "../../../../../components/core/typography/Link";
+
+type Props = {
+  openTosModal: () => void;
+  onSearch: () => void;
+  methodType: "bancomatpay" | "pagobancomat";
+};
+
+type LocalesText = {
+  title: string;
+  text1: string;
+  text2: string;
+  text3: string;
+  text4: string;
+};
+
+const bancomatLocales = {
+  title: I18n.t("wallet.searchAbi.bancomat.title"),
+  text1: I18n.t("wallet.searchAbi.bancomat.description.text1"),
+  text2: I18n.t("wallet.searchAbi.bancomat.description.text2"),
+  text3: I18n.t("wallet.searchAbi.bancomat.description.text3"),
+  text4: I18n.t("wallet.searchAbi.bancomat.description.text4")
+};
+
+const mapLoacalesFromMethod = new Map<string, LocalesText>([
+  [
+    "bancomatpay",
+    {
+      title: I18n.t("wallet.searchAbi.bpay.title"),
+      text1: I18n.t("wallet.searchAbi.bpay.description.text1"),
+      text2: I18n.t("wallet.searchAbi.bpay.description.text2"),
+      text3: I18n.t("wallet.searchAbi.bpay.description.text3"),
+      text4: I18n.t("wallet.searchAbi.bpay.description.text4")
+    }
+  ],
+  ["bancomat", bancomatLocales]
+]);
+
+export const SearchStartComponent: React.FunctionComponent<Props> = (
+  props: Props
+) => {
+  const locales = fromNullable(
+    mapLoacalesFromMethod.get(props.methodType)
+  ).getOrElse(bancomatLocales);
```

Comment:
With the current implementation the default language is always used because is not calculated at rendering time.

<img src=https://user-images.githubusercontent.com/26501317/104013490-cae50500-51b1-11eb-92c4-c1792fb00791.png width=300 />

One possible solution:

```suggestion
const bancomatLocales = () => ({
  title: I18n.t("wallet.searchAbi.bancomat.title"),
  text1: I18n.t("wallet.searchAbi.bancomat.description.text1"),
  text2: I18n.t("wallet.searchAbi.bancomat.description.text2"),
  text3: I18n.t("wallet.searchAbi.bancomat.description.text3"),
  text4: I18n.t("wallet.searchAbi.bancomat.description.text4")
});

const bancomatPayLocales = () => ({
  title: I18n.t("wallet.searchAbi.bpay.title"),
  text1: I18n.t("wallet.searchAbi.bpay.description.text1"),
  text2: I18n.t("wallet.searchAbi.bpay.description.text2"),
  text3: I18n.t("wallet.searchAbi.bpay.description.text3"),
  text4: I18n.t("wallet.searchAbi.bpay.description.text4")
});

const loadLocales = (methodType: "bancomatpay" | "pagobancomat") =>
  methodType === "bancomatpay" ? bancomatPayLocales() : bancomatLocales();

export const SearchStartComponent: React.FunctionComponent<Props> = (
  props: Props
) => {
  const locales = loadLocales(props.methodType);
```